logic.js

```diff
@@ -0,0 +1,27 @@
+class Logic{
+
+    checkMatch(card1, card2){
+        if($(card1).data().id === $(card2).data().id && $(card1).data().gridArea !== $(card2).data().gridArea){
```

Comment:
The statement inside the `if` returns true or false so you can actually return the statement itself with no need for the `if`